documentation/modules/rn-2.5.adoc

```diff
@@ -124,40 +130,113 @@ When migrating VMs that are installed with RHEL 9 as guest operating system from
+
+For more information, see link:https://access.redhat.com/security/cve/CVE-2023-39321[CVE-2023-39321].
+
+
+.CVE-2023-39319: Flaw in html/template package
+
+A flaw was found in the Golang `html/template` package used in {project-short}. This flaw means {project-short} versions before {project-short} 2.5.3 are vulnerable, as the `html/template` package did not properly handle occurrences of `<script`, `<!--`, and `</script` within JavaScript literals in `<script>` contexts. This flaw could cause the template parser to improperly consider script contexts to be terminated early, causing actions to be improperly escaped, which could be leveraged to perform an `XSS` attack. link:https://issues.redhat.com/browse/MTV-693[(MTV-693)]
+
+This issue has been resolved in {project-short} 2.5.3. It is advised to update to this version of {project-short} or later.
+
+For more information, see link:https://access.redhat.com/security/cve/CVE-2023-39319[CVE-2023-39319].
+
+
+.CVE-2023-39318: Flaw in html/template package
+
+A flaw was found in the Golang `html/template` package used in {project-short}. This flaw means {project-short} versions before {project-short} 2.5.3 are vulnerable as the `html/template` package did not properly handle HMTL-like `""` comment tokens, nor hashbang `\#!` comment tokens. This flaw could cause the template parser to improperly interpret the contents of `<script>` contexts, causing actions to be improperly escaped, which could be leveraged to perform an `XSS` attack. link:https://issues.redhat.com/browse/MTV-693[(MTV-693)]
+
+This issue has been resolved in {project-short} 2.5.3. It is advised to update to this version of {project-short} or later.
+
+For more information, see link:https://access.redhat.com/security/cve/CVE-2023-39318[CVE-2023-39318].
+
+
+.Logs archive file downloaded from UI includes logs related to deleted migration plan/VM
+
+In previous releases of {project-short} {project-version}, the log files downloaded from UI could contain logs that are related to a previous migration plan. link:https://issues.redhat.com/browse/MTV-783[(MTV-783)]
+
+This issue has been resolved in {project-short} 2.5.3.
+
+
+.Extending a VM disk in RHV is not reflected in the MTV inventory
+
+In previous releases of {project-short} {project-version}, the size of disks that are extended in RHV was not adequately monitored. This resulted in the inability to migrate virtual machines with extended disks from a RHV provider. link:https://issues.redhat.com/browse/MTV-830[(MTV-830)]
+
+This issue has been resolved in {project-short} 2.5.3.
+
+
+.Filesystem overhead configurable
+
+In previous releases of {project-short} {project-version}, the filesystem overhead for new persistent volumes was hard-coded to 10%. The overhead was insufficient for certain filesystem types, resulting in failures during cold-migrations from RHV and OSP to the cluster where {project-short} is deployed.
```

Comment:
as I commented on #447, it works both ways - to increase the overhead as written here, and also to reduce the overhead to minimize unused storage consumption (which is what MTV-699 refers to)